addons/nuxeo-retention/nuxeo-retention-core/src/test/java/org/nuxeo/retention/test/RetentionTestCase.java

```diff
@@ -108,7 +110,7 @@ protected void awaitRetentionExpiration(long millis) throws InterruptedException
 
     protected RetentionRule createRuleWithActions(RetentionRule.ApplicationPolicy policy,
             StartingPointPolicy startingPointPolicy, List<String> docTypes, String startingPointEventId,
-            String startingPointExpression, String matadataXPath, long years, long months, long days,
+            String startingPointExpressionLanguage, String startingPointEventInput, String matadataXPath, long years, long months, long days,
```

Comment:
small typo: `matadataXPath`

Reply:
thanks, I will do it on master with a separate commit `cleanup/format`